### ISSUE TITLE
Notebook crashes on large data

### ISSUE THREAD

**Issue by rohailiqbal** [microsoft/azuredatastudio#6543]
Issue Type: <b>Bug</b>

SQL Server
I have **large database** with millions of record in many tables.
1. click +Code
2. select * from table1
it will show top 5000 rows
3. Click +text
type something (Azure Data Studio will start pausing and late response)
4. Click +Code
5. select * from table2
it will try loading data... keeps on trying and crashes

Azure Data Studio version: azuredatastudio 1.9.0 (78a42e1d112ae3231777722b51eaf44f83ddbe55, 2019-07-10T04:31:36.998Z)
OS version: Windows_NT x64 6.1.7601

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-4570 CPU @ 3.20GHz (4 x 3192)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.91GB (5.05GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by chlafreniere** [comment]
Thanks, @rohailiqbal. It may be related to #6392 but let's keep this open for now in case the culprit is different.

**Comment by chlafreniere** [comment]
We've spent some significant effort improving memory usage/performance for notebooks in the September release (current efforts can be seen in insiders). I tested a large (~60MB) notebook on both with the August (1.10) release and our current insiders release. Here's the difference:

August:
![image](https://user-images.githubusercontent.com/40371649/63893790-eb873300-c99f-11e9-92e4-fc3b9ab5bc04.png)

Insiders [i.e. September release soon]:
![image](https://user-images.githubusercontent.com/40371649/63893800-ef1aba00-c99f-11e9-98ba-26ab911e2c5f.png)

If you end up seeing any issues in the future with this, I would love to hear about it, as the team put significant effort into minimizing any crashes as much as posible.
